electricpy/__init__.py

```diff
@@ -4465,10 +4470,59 @@ def tap_changing_transformer(v1, v2, p, q, r, x):
         
     '''
 
-    ts = (v2 / v1) * (1 / (1 - (r * p + x * q) / (v1 * v2)))
+    ts = (Vgen*Vgen) / (Vgen*Vdis - (R * Pload + X * Qload) )
 
     return pow(ts, 0.5)
 
+def suspension_insulators(number_capacitors, capacitance_ratio, Voltage):
+    '''
+        This function calculates the voltage across each capacitor in a suspension insulator strain
+
+        reference:https://electrical-engineering-portal.com/download-center/books-and-guides/power-substations/insulator-pollution  
+
+
+        Parameters
+        ----------
+        
+        number_capacitors: number of disk capacitors hung to transmission line
+        
+        capacitance_ratio: ratio of disk capacitance and pin to pole air capacitance
+        
+        Voltage: voltage to the wire
```

Comment:
Where is this voltage measured? I don't think that's clear, but we should probably specify.

Reply:
completed in 9e81883ef1b4b12f6e1d23e2badc681666b68951